src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -115,6 +117,7 @@ public class ScriptHost : JobHost, IScriptJobHost
             FunctionErrors = new Dictionary<string, ICollection<string>>(StringComparer.OrdinalIgnoreCase);
             EventManager = eventManager;
             _functionDispatcher = functionDispatcher;
+            _languageWorkerChannelManager = languageWorkerChannelManager;
```

Comment:
I don't think ScriptHost should have a dependency on this manager. It already has a dependency on IFunctionDispatcher and the dispatcher is responsible for managing channels for the specialized host.

Reply:
Makes sense! Moved this into FunctionDispatcher